src/test/java/com/teragrep/pth10/steps/teragrep/bloomfilter/BloomFilterTableTest.java

```diff
@@ -60,7 +60,7 @@ class BloomFilterTableTest {
 
     final String username = "sa";
     final String password = "";
-    final String connectionUrl = "jdbc:h2:~/test;MODE=MariaDB;DATABASE_TO_LOWER=TRUE;CASE_INSENSITIVE_IDENTIFIERS=TRUE";
+    final String connectionUrl = "jdbc:h2:mem:test;MODE=MariaDB;DATABASE_TO_LOWER=TRUE;CASE_INSENSITIVE_IDENTIFIERS=TRUE";
```

Comment:
This changes h2 engine to use in memory database instead of writing to home directory. This is better for unit tests since the data does not need to persist after the test.